cluster-operator/src/test/java/io/strimzi/operator/cluster/model/ListenersValidatorTest.java

```diff
@@ -1126,4 +1127,49 @@ public void testValidateCreateBootstrapService() {
                 "listener listener2 cannot configure createBootstrapService because it is not load balancer listener",
                 "listener listener3 cannot configure createBootstrapService because it is not load balancer listener"));
     }
+    
+    @ParallelTest
+    public void testValidateBootstrapExternalIPsListenener() {
```

Comment:
Yu should probbaly add a third listener with node prot type and see that it doesn't raise any errors.